rocketpy/simulation/flight.py

```diff
@@ -3357,7 +3434,11 @@ class TimeNodes:
         TimeNodes object are instances of the TimeNode class.
         """
 
-        def __init__(self, init_list=[]):
+        # pylint: disable=missing-function-docstring
+
+        def __init__(self, init_list=None):
+            if not init_list:
+                init_list = []
```

Comment:
This is in the exceptions already

Reply:
[2dae970](https://github.com/RocketPy-Team/RocketPy/pull/621/commits/2dae970679dd2ddbd2a6189680dd8e89f9666379)